front/components/Avatar/avatar.spec.tsx

```diff
@@ -0,0 +1,42 @@
+import { render, screen } from '@testing-library/react';
+
+import Avatar from './index';
+
+const mockupUser = {
+  id: '414',
+  username: 'Toto',
+  email: 'toto@gmail.com',
+  description: "I'm an example",
+  packages: [],
+};
```

Comment:
We probably want to generate this kind of fake data using `faker` or something like [`test-data-bot`](https://github.com/jackfranklin/test-data-bot)